docs/workflow.md

```diff
@@ -0,0 +1,30 @@
+# Workflow
+
+## Branch naming
+
+Please follow this naming pattern:
+
+- feature => f/#issuenumer-small-description
+- issue => b/#issuenumber-small-description
+
+## Release
+
+We are working with github projects. Our next branch will always the next release incoming.
+
+## Process
+
+When you will create your merge request. Your branch will have to validate these conditions:
+
+- Respect the [ISSUE_TEMPLATE](../.github/ISSUE_TEMPLATE/bug_report.md) or [FEATURE_TEMPLATE](../.github/ISSUE_TEMPLATE/feature_request.md)
+- Have a coherent branch name
+- Validate eslint
+- Validate our chromatic visual non regression
```

Comment:
gonna need more explanations about this step

Reply:
i wanted to give the basic process from the starting point to the pull request. What do yout hink can be the best here ?